algo/src/main/kotlin/com/hartwig/actin/algo/evaluation/treatment/HasOnlyHadTreatmentWithCategoryOfTypes.kt

```diff
@@ -12,9 +12,7 @@ class HasOnlyHadTreatmentWithCategoryOfTypes(
     private val types: Set<TreatmentType>
 ) : EvaluationFunction {
     override fun evaluate(record: PatientRecord): Evaluation {
-        val treatmentsByMatchEvaluation = record.oncologicalHistory.flatMap { it.allTreatments() }.filter {
-            it.categories().intersect( TreatmentCategory.SYSTEMIC_CANCER_TREATMENT_CATEGORIES ).isNotEmpty()
-        }.groupBy {
+        val treatmentsByMatchEvaluation = record.oncologicalHistory.flatMap { it.allTreatments() }.filter { it.isSystemic }.groupBy {
```

Comment:
With a long call chain like this, I think it helps to put each function call on its own line:
```suggestion
        val treatmentsByMatchEvaluation = record.oncologicalHistory.flatMap { it.allTreatments() }
            .filter { it.isSystemic }
            .groupBy {
```